docs/src/main/sphinx/release/release-407.md

```diff
@@ -0,0 +1,104 @@
+# Release 407 (13 Feb 2023)
+
+## General
+
+* Add support for adaptive task request sizing. It can be enabled with the
+  `query.remote-task.enable-adaptive-request-size` configuration property and
+  configured with the `query.remote-task.max-request-size`,
+  `query.remote-task.request-size-headroom`, and
+  `query.remote-task.guaranteed-splits-per-task` configuration properties or
+  their respective session properties. ({issue}`15721`)
+* Add support for correlated queries with single-row values. ({issue}`15989`)
+* Add support for configuring the minimum number of rows and
+  minimum data size per task using the `optimizer.min-input-rows-per-task` and
+  `optimizer.min-input-size-per-task` configuration properties. ({issue}`15489`)
+* Improve concurrency when small, concurrent queries are run on a large cluster
+  by automatically determining how many nodes to use for distributed joins
+  and aggregations. This can be managed with the `query.max-hash-partition-count`
+  (renamed from `query.hash-partition-count`) and `query.min-hash-partition-count`
+  configuration properties. ({issue}`15489`)
+* Improve query memory tracking. ({issue}`15983`)
+* Improve query performance when a predicate evaluates to a null value. ({issue}`15744`)
+* Improve performance of queries with joins on the output of global
+  aggregations. ({issue}`15858`)
+* Improve performance of selective queries, queries that read a small number of
+  columns, and queries that process tables with large Parquet row groups or Orc
+  stripes. ({issue}`15579`)
+* Improve performance of queries with window functions. ({issue}`15994`)
+* Change the `table_type` of materialized views in `information_schema.tables`
```

Comment:
@findepi , why did this change? We discussed this being incompatible with the SQL spec, so it shouldn't have gone in.

Reply:
Can we revert to unblock the release, then discuss this later?

Reply:
I agree with reverting for now so we can get a release out.

Reply:
cc @jhlodin